### PR TITLE
[ioctl] Dev account info

### LINKED ISSUE #2936: [ioctl] build `account info` command line into new ioctl
The new ioctl (./ioctl/newcmd) is a refactored version of the old ioctl(./ioctl/cmd). It includes two major changes:
- The [Client interface](https://github.com/iotexproject/iotex-core/blob/master/ioctl/client.go) is passed when constructing the Cobra command.
- Unit tests are added for each command.

The code is a draft. The code of old ioctl can be used for reference.

**What would you like to be added**:
add `accountinfo.go` and `accountinfo_test.go` in `./ioctl/newcmd/account`. 
**Code for reference**:
https://github.com/iotexproject/iotex-core/blob/master/ioctl/cmd/account/accountinfo.go

### PR DESCRIPTION
**Pull request by LuckyPigeon**
fix #2936, #3109

### CODE REVIEW

File: ioctl/newcmd/account/accountinfo.go [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+			}
+			balance := big.NewInt(0)
+			if accountMeta.Balance != "" {
+				var ok bool
+				balance, ok = new(big.Int).SetString(accountMeta.Balance, 10)
+				if !ok {
+					return output.NewError(output.ConvertError, "failed to set account balance", err)
+				}
+			}
+			ethAddr, err := address.FromString(addr)
```
Comment (CoderZhi) [inline ioctl/newcmd/account/accountinfo.go:66]: this is not an eth address

File: ioctl/newcmd/account/accountinfo.go [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+				EthAddress:       ethAddr.Hex(),
+				Balance:          util.RauToString(balance, util.IotxDecimalNum),
+				Nonce:            int(accountMeta.Nonce),
+				PendingNonce:     int(accountMeta.PendingNonce),
+				NumActions:       int(accountMeta.NumActions),
+				IsContract:       accountMeta.IsContract,
+				ContractByteCode: hex.EncodeToString(accountMeta.ContractByteCode),
+			}
+
+			fmt.Println(message.String())
```
Comment (CoderZhi) [inline ioctl/newcmd/account/accountinfo.go:81]: client.Print

**Review by CoderZhi — COMMENTED** [review]
@huof6829 please review this PR

**Comment by huof6829** [comment]
add sub command in NewAccountCmd of account.go

**Review by CoderZhi — APPROVED** [review]

File: ioctl/newcmd/account/accountinfo_test.go [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+	}}
+
+	t.Run("retrieve account information successfully", func(t *testing.T) {
+		client.EXPECT().APIServiceClient(gomock.Any()).Return(apiServiceClient, nil)
+		client.EXPECT().PrintInfo(gomock.Any()).Do(func(info string) {
+			fmt.Println(info)
```
Comment (CoderZhi) [inline ioctl/newcmd/account/accountinfo_test.go:51]: delete

File: ioctl/newcmd/account/account.go [hunk 4]
```diff
@@ -179,7 +180,7 @@ func PrivateKeyFromSigner(client ioctl.Client, signer, password string) (crypto.
 }
 
 // GetAccountMeta gets account metadata
-func GetAccountMeta(addr string, client ioctl.Client) (*iotextypes.AccountMeta, error) {
+func GetAccountMeta(client ioctl.Client, addr string) (*iotextypes.AccountMeta, error) {
```
Comment (CoderZhi) [inline ioctl/newcmd/account/account.go:183]: rename to `AccountMeta`

File: ioctl/newcmd/account/accountinfo_test.go (hunk 3 above)
Comment (LuckyPigeon) [reply ioctl/newcmd/account/accountinfo_test.go:51]: Just a simulation for `client.PrintInfo` in `accountinfo.go` L92

Comment (CoderZhi) [reply ioctl/newcmd/account/accountinfo_test.go:51]: Not needed in unit test

**Comment by huof6829** [comment]
@LuckyPigeon  format is useless, so delete

**Review by huof6829 — APPROVED** [review]

**Pull request merged**